### DISCUSSION TITLE
msgpackrpc.error.RPCError: rpclib: client error C0002: Function 'simGetImages' was called with an invalid number of arguments. Expected: 2, got: 3

### DISCUSSION THREAD

**Discussion by BazilaAfridi** [microsoft/AirSim#4455] [Support Q&A]
![image](https://user-images.githubusercontent.com/69035494/161503511-94b105ab-3095-44b6-baf1-191bb2d587cb.png)
Can anyone please guide me on why I'm having this issue?

**Comment by zimmy87** [comment]
Looks like there's a client/server mismatch here. Your python client has #3320 applied but not your server. I'm assuming the server is a PEDRA environment? I'm not very familiar with this project or what version of AirSim it is based on, but I think you may need to downgrade your python client by doing the following:

`pip install airsim==1.5.0`

You may need to experiment with the specific version, but 1.5.0 looks like the latest verion without #3320.

**Comment by zclyaya** [comment]
> ![image](https://user-images.githubusercontent.com/69035494/161503511-94b105ab-3095-44b6-baf1-191bb2d587cb.png) Can anyone please guide me on why I'm having this issue?

hi ,I meet the same problem, can you tell me how to solve it?

**Reply by zclyaya** [reply]
sorry ，I did not solve this problem,do you have  a solution?

**Reply by Echodeep** [reply]
sorry ，I did not solve this problem,too. Do you have a solution?
